ReactiveXComponent/Connection/IXCSubscriber.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using ReactiveXComponent.RabbitMq;
+using ReactiveXComponent.RabbitMQ;
+
+namespace ReactiveXComponent.Connection
+{
+    public interface IXCSubscriber : IDisposable
```

Comment:
I should have access to an observable collection
add a method called getStateMachineUpdates for example